scripts/startup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright (C) 2018 Expedia Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+SHUNTING_YARD_HOME=/opt/shunting-yard
+
+[[ ! -z $SHUNTINGYARD_CONFIG_YAML ]] && echo "$SHUNTINGYARD_CONFIG_YAML"|base64 -d > ${SHUNTING_YARD_HOME}/conf/shunting-yard-config.yml
+
+[[ -z $HEAPSIZE ]] && export HEAPSIZE=1024
+
+core-site-template() {
```

Comment:
If you don't have need to change this file during runtime, can you just create static file during docker build?

Reply:
done.